Documentation/operations/upgrade.rst

```diff
@@ -299,6 +299,8 @@ communicating via the proxy must reconnect to re-establish connections.
   ``spec.transport.localPort`` in ``CiliumBGPPeerConfig`` has been removed and will be ignored if it was configured in the ``v2alpha1`` version.
 * The ``CiliumBGPPeeringPolicy`` CRD is deprecated and will be removed in a future release. Please migrate to ``cilium.io/v2``
   BGP CRDs (``CiliumBGPClusterConfig``, ``CiliumBGPPeerConfig``, ``CiliumBGPAdvertisement``, ``CiliumBGPNodeConfigOverride``) to configure BGP.
+* The label ``io.cilium.k8s.policy.serviceaccount`` will be included in the default label list. If you configure your own Identity-Relevant Labels 
```

Comment:
Can you test and add information how to **not** enable new default behavior please? I believe `!k8s:io\\.cilium\\.k8s\\.policy\\.serviceaccount` should work. Some thing like: "If you would like to disable new behavior, you can add following label filter to disable it: ..."